### PR TITLE
[serve] Make fractional resource usage more obvious in docs

### LINKED ISSUE #11574: [Serve] [Doc] Add a section for fractional CPU/GPU
### Describe your feature request
https://ray-distributed.slack.com/archives/CNCKBBRJL/p1603439306100200

### PR DESCRIPTION
**Pull request by edoakes**
## Why are these changes needed?

Users aren't clear on how they can split resources across backend replicas.

## Related issue number

Closes https://github.com/ray-project/ray/issues/11574

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by simon-mo** [comment]
can you add a sentence on multiplexing CPUs as well? Several users has asked about that already.

File: doc/source/serve/advanced.rst [hunk 1]
```diff
@@ -44,6 +45,19 @@ following:
...
+--------------------
+
+The resources specified in ``ray_actor_options`` can also be *fractional*.
+This allows you to flexibly share resources between workers.
+For example, if you have two models and each doesn't fully saturate a GPU, you might want to have them share a GPU by allocating 0.5 GPUs each:
+
+.. code-block:: python
+
+  half_gpu_config = {"num_gpus": 0.5}
+  client.create_backend("my_gpu_backend_1", handle_request, ray_actor_options=config)
```
Comment (rkooo567) [inline doc/source/serve/advanced.rst:58]: Isn't it supposed to be `half_gpu_config` instead of `config`?

Comment (edoakes) [reply doc/source/serve/advanced.rst]: yeah good catch

**Comment by edoakes** [comment]
@simon-mo added a sentence, PTAL

**Review by simon-mo — APPROVED** [review]

**Pull request merged**